Refresher/UI/PipelineForm.cs

```diff
@@ -481,4 +482,10 @@ private void OnLog(RefresherLog log)
             }
         });
     }
+
+    protected override void OnClosing(CancelEventArgs e)
+    {
+        Environment.Exit(0);
```

Comment:
This is a hack rather than a fix.

It would be preferrable to find out why `App.Run` in `Program` doesn't respond. Probably something to do with child forms.